### ISSUE TITLE
create a text block area

### ISSUE THREAD

**Issue by vyscond** [reingart/pyfpdf#28]
I've trying to print a table which can support multiple lines over the heads or the content itself. The problem is that the multi_cell function only accept the height of the cells, so we cannot adjust the height size equally to all the multi_cells side by side. Like this example:

```
+---------------+---------------+----------------+
|               | one line text |                |
| Two line      +---------------+ Two line       |
| Text          |               | text           |
|               |               |                |
+---------------+               +----------------+
```

So we manage the columns (multi_cell) to have the same height per cell, but not the same area height which makes the usage of full border "incorrect". It can be used only the left and right border and then draw manually the top and bottom lines but the alignment of the text will be a problem. the one line of text will not be vertically aligned.

Basically the multi_cell cannot be used as a text box area. So i think the library is missing this: a text box area. I know this is not easy to implement, but i don't see any topic around here related to tries. So there is any plan to build this type of "widget", or it's just another of "shut up and find out other library"?

Well this project by far is the best i have found. It's pretty complete and easy to use. But has the same problem from the other: missing a text box area. If can get this the translation of html table, and the creation of table by template will be easy and less buggy.
